app/views/shared/_header.html.haml

```diff
@@ -1,5 +1,6 @@
 .navbar.navbar-static-top
-  .navbar-inner
+  - styles = (Rails.env == "stage" && Settings.stage_color.present?) ? "background-image: none; background-color:  #{Settings.stage_color}" : ""
```

Comment:
```suggestion
  - apply_stage_color = Rails.env.staging? && Settings.stage_color.present?
  - stage_style = apply_stage_color ? "background-image: none; background-color:  #{Settings.stage_color}" : ""
```

Reply:
It appears that the env is `stage` not `staging`, so it would be `Rails.env.stage?`, not `Rails.env.staging?`

Reply:
@giladshanan @rdunlop I added a helper method to make the view a bit cleaner https://github.com/tablexi/nucore-open/pull/3015/files#diff-5f3fc5e3977d242572aa1d08551f5eb557de0ccaff30370838ee9df5386ea0daR23